### PR TITLE
Creation infos page

### PR DESCRIPTION
**Pull request by remi59800**

### CODE REVIEW

**Review by ThomasDCW — DISMISSED** [review]
ok

File: frontend/src/pages/Infos/index.jsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      text: "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Lorem ipsum dolor sit amet, consectetur. Version 1.0.0",
+    },
+  ];
+  return (
+    <SAccordion>
+      <div className="infos-container">
+        <Link
+          className="button-return"
+          to="/profil/:id"
+          style={{ textDecoration: "none" }}
```
Comment (LuigiMenez) [inline frontend/src/pages/Infos/index.jsx:31]: Why, the style of the Link, is not in the style file?

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

**Review by LoicBrassart — APPROVED** [review]
If you needed to "fix eslint", it means you bypassed it the first time around ? -_-

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

**Pull request merged**